bin/run_faucet

```diff
@@ -17,6 +17,7 @@ docker_volumes=
 port_opts=
 local=
 faucet_id=
+fot=
```

Comment:
I think this needs to be a new container ("controller") not an extension of faucet.  You can start with the faucet one and add more stuff to it, but it's not "faucet" anymore.  Inside of this controller container would be { faucet, forch, gauge, daq, XXX } -- rather than (e.g. what you have now) one container that's { faucet, forch} and another one that's { gauge }. Basically, the new "controller" container should be an encapsulation of what is running on the controller.

Reply:
That sounds liek a lot of code duplication though. An alternative could be to rename current scripts to controller and let the flag decide whether to add daq and forch into it.

Reply:
Separation done.